sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/aio/operations/_operations.py

```diff
@@ -415,6 +431,7 @@ async def register(  # pylint: disable=inconsistent-return-statements
             401: ClientAuthenticationError,
             404: ResourceNotFoundError,
             409: ResourceExistsError,
+            304: ResourceNotModifiedError,
```

Comment:
Is this a GET call? If not, why are we expecting a 304 answer? Is this just "let's apply the same map everywhere" thing?

Reply:
No, this is a [PUT call](https://github.com/Azure/azure-rest-api-specs/blob/4b2f5a45a34e351e38e8b075cf1da5a775fd414a/specification/schemaregistry/data-plane/Microsoft.EventHub/stable/2021-10/schemaregistry.json#L385). According to Izzy and [[this issue](https://github.com/Azure/autorest.python/issues/1415)], this is added by default everywhere and doesn't take input from the swagger.